apps/nextjs/src/env.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-restricted-properties */
```

Comment:
this disable directive should still be required given we're referencing `process.env` in this file which should not be allowed with our custom lint config